### PR TITLE
bold keywords in readme

### PR DESCRIPTION
**Pull request by qpwo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,20 +10,20 @@
 
 Bonnie is a command aliasing tool that allows you to quickly and efficiently define short aliases for long commands that you have to repeatedly run. Here's a quick feature overview:
 
--   ✨ Supports simple key-value aliasing
--   ✨ Supports inserting custom arguments into commands
--   ✨ Supports interpolating environment variables
--   ✨ Supports adding any and all arguments given into a single place
+-   ✨ Supports simple **key-value aliasing**
+-   ✨ Supports inserting **custom arguments** into commands
+-   ✨ Supports **interpolating** environment variables
```
Comment (arctic-hen7) [inline README.md:15]:
```suggestion
-   ✨ Supports **interpolating environment variables**
```

**Review by arctic-hen7 — CHANGES_REQUESTED** [review]
This looks great, thanks! Just one very minor thing and then we can merge.

**Comment by qpwo** [comment]
Hmm I think I committed the suggestion but it's still saying "1 change requested" but I can't approve it --- anyway of course change it however you'd like it's your repo

**Review by arctic-hen7 — APPROVED** [review]
Yep, I just have to re-review it. And now I have!

**Pull request merged**